static/app/components/forms/model.tsx

```diff
@@ -11,7 +11,7 @@ import {defined} from 'sentry/utils';
 type Snapshot = Map<string, FieldValue>;
 type SaveSnapshot = (() => number) | null;
 
-export type FieldValue = string | number | boolean | Choice | undefined; // is undefined valid here?
+export type FieldValue = string | number | boolean | Choice | Set<string> | undefined; // is undefined valid here?
```

Comment:
@roggenkemper we don't want a set either, pretty sure that will cause problems in Amplitude

Reply:
this is an independent update to the FieldValue type, since it can be a Set but it wasn't. in the analytics call, anything that is a set will be transformed into a comma separated string